Chat/ClientApp/src/containers/AttendeesArea.ts

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import AttendeesArea from '../components/AttendeesArea';
+import { State } from '../core/reducers';
+
+const mapStateToProps = (state: State) => ({
+  eventAttendees: state.threadMembers.threadMembers
```

Comment:
Why do you need .threadMembers.threadMembers here?

Reply:
I agree its an odd name but it is just the chain of properties to get all the members in the chat